about.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Microverse</title>
+    <link rel="stylesheet" href="./styles/about/about.css">
+    <link rel="shortcut icon" href="./images/microverse.png" type="image/x-icon">
+</head>
+<body>
+    <main class="container">
+        <header class="container__header">
+            <nav class="hamburger-menu">
+                <img src="images/hamburger-menu.png" alt="hamburger menu icon">
+            </nav>
+        </header>
+
+        <section class="introduction">
+            <h1 class="introduction__header">
+                Microverse Full-Stack web development program
+            </h1>
+            <div class="introduction__description-container">
+                <p class="introduction__description">
+                    At Microverse, students learn through a very unique approach by pair programming and collaborating with other students in real-time, just as they would with colleagues of a distributed team in a real company.
+                    Students learn multiple programming languages and Computer Science fundamentals, through working on real projects with other students around the world.
+                </p>
+            </div>
+            <p class="contact-us-email">
+                Please contact us per Email for any further questions about Microverse program
+                <a href="mailto:admissions@microverse.org">admissions@microverse.org</a>
+            </p>
+        </section>
+
+        <section class="alumni-opinion">
+            <h3>
+                What do alumni think of Microverse?
+            </h3>
+            <p>
+                Microverse accepted its first cohort in early 2018 and its first graduates started receiving job offers in September 2018. Here are some of their comments, reviews, and videos. 
+            </p>
+
+            <div class="img-container">
+                <img src="images/canda.png" alt="Canada flag">
+                <div class="img-overlay">
```

Comment:
- [ ] Please, for all the project try to use the colors according the template. 🥥 
- Basic color swatch: #272A31, #EC5242, #D3D3D3